docs/clojure/anonymous-function.md

```diff
@@ -96,10 +96,11 @@ If we use def to save the value...
 
 `combined` is exposed outside of our function, and this is bug-prone.
 
-~~~klipse
```

Comment:
Aha, github doesn't render html markups, so I wondered what you mentioned about. But looking at raw markdown, I figured out. :)
I think your change, using data-eval-context="expr", works well. The doc makes sense.
I'll add this kind of info to the "How to Contribute", http://clojurebridge.github.io/community-docs/docs/contribute/ . When I edit, I should consider this.